packages/amplify_datastore/ios/Classes/DataStoreHubEventStreamHandler.swift

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Foundation
+import Amplify
+import AmplifyPlugins
+
+public class DataStoreHubEventStreamHandler: NSObject, FlutterStreamHandler {
+
+    private var eventSink: FlutterEventSink?
+    private var token: UnsubscribeToken?
+    private var flutterModelRegistration: FlutterModels?
+    
+    public func registerModelsForHub(flutterModels: FlutterModels) {
+        self.flutterModelRegistration = flutterModels
+    }
+
+    public func onListen(withArguments arguments: Any?, eventSink events: @escaping FlutterEventSink) -> FlutterError? {
+        self.eventSink = events
+        setToken()
+        return nil
+    }
+    
+    public func setToken() {
```

Comment:
nit: Can we remove the public visibility of this func and maybe rename to `subscribeToHubEvents` or something like that?

Reply:
Changing this to "setHubListener".  Changed the visibility as requested.